libs/domains/service-helm/feature/src/lib/hooks/use-kubernetes-services/use-kubernetes-services.tsx

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseKubernetesServicesProps {
+  helmId: string
+}
+
+export function useKubernetesServices({ helmId }: UseKubernetesServicesProps) {
```

Comment:
Perhaps it should be better to have `Services` before? Like we have in the domains `service`? 

`useServicesKubernetes`

Reply:
Nope this is the correct english spelling for what it represents.
Here we list services with a kubernetes "attribute"